### PR TITLE
ECOM-4470 update orders api response to include user information

### PR DESCRIPTION
**Pull request by PaulWattenberger**
Needed to capture historical enroll/purchase information

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PaulWattenberger! I've created [OSPR-1265](https://openedx.atlassian.net/browse/OSPR-1265) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](http://open.edx.org/sites/default/files/wysiwyg/individual-contributor-agreement.pdf)     or indicated your institutional affiliation.     If you like, you can add yourself to the [AUTHORS](https://github.com/edx/ecommerce/blob/master/AUTHORS)      file for this repo, though that isn't required.     Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

File: ecommerce/extensions/api/v2/tests/views/test_orders.py [hunk 1]
```diff
@@ -104,6 +104,17 @@ def test_staff_superuser(self, is_staff, is_superuser):
...
+        """ Make sure that the correct user information is returned. """
+        admin_user = self.create_user(is_staff=True, is_superuser=True)
+        order = factories.create_order(user=admin_user)
+
+        response = self.client.get(self.path, HTTP_AUTHORIZATION=self.generate_jwt_token_header(admin_user))
+        content = json.loads(response.content)
+        self.assertEqual(content['count'], 1)
+        self.assertEqual(content['results'][0]['number'], unicode(order.number))
+        self.assertEqual(content['results'][0]['user']['email'], admin_user.email)
+
```
Comment (clintonb) [inline ecommerce/extensions/api/v2/tests/views/test_orders.py:118]: What about the username field?

**Comment by clintonb** [comment]
👍 (once branch updated and commits squashed into a single commit)

**Comment by clintonb** [comment]
You did squash your commits.

**Comment by clintonb** [comment]
Also, remember to delete your branches after merging.

**Comment by nedbat** [comment]
Hey guys! The bot thinks @PaulWattenberger is an outside contributor.  We should get him set up in the database so the bot understands.  There must have been a step missed in the onboarding?

**Pull request merged**